instructure/canvas-ios
# Add logging in with QR code

refs: MBL-14110
affects: student
release note: Added the ability to log in with a QR code

Test plan:
- Log into web at mobiledev.beta.instructure.com > hodor (ask me for password)
- Global Nav > Account > QR for Mobile Login
- Launch the Student App and log out or change users
- Tap 'QR Login'
- Tap Next on Tutorial
- Scan the QR code from web
- Should get logged in
- Scan a QR code that isn't a login code
- Should get an error
